tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -26,7 +26,7 @@ h1, h2, h3, h1 span, h2 span, h3 span, .nav-item {
   font-family: $font-family-title;
 }
 
-#content.content-wrapper {
+header.global-header ~ #content.content-wrapper {
```

Comment:
OK now that I understand, I'd like to suggest a better selector. Can we write instead `:not(.courseware) > #content.content-wrapper` to make it clear that the rule should only apply outside of the courseware iframe? Also please add a comment to the css rule such that I remember lated why we created it.